.amethyst.sample.yml

```diff
@@ -1,42 +1,280 @@
-# Use tall and fullscreen layouts
+# Default settings for amethyst
+# Repo: `https://github.com/ianyh/Amethyst`
+#
+# Move this file to: `~/.amethyst.yml`
+# In order to register changes restart Amethyst.
+# If you experience issues pulling in the changes you can also quit Amethyst and run: `rm ~/Library/Preferences/com.amethyst.Amethyst.plist`
+# This removes the current preferences and causes a re-load of this file when amethyst restarts.
+
+# layouts	Ordered list of layouts to use by layout key (default tall, wide, fullscreen, and column).
 layouts:
-    - tall
-    - fullscreen
-    - tall-right
-    - wide
-    - two-pane
-    - 3column-left
-    - middle-wide
-    - 3column-right
-    - 4column-left
-    - 4column-right
-    - column
-    - row
-    - floating
-    - widescreen-tall
-    - widescreen-tall-right
-    - bsp
-
-# Add "command" to the default mods
+  - tall
+  - fullscreen
+  # - tall-right
+  - wide
+  # - two-pane
+  # - middle-wide
+  # - 3column-left
+  # - 3column-middle
+  # - 3column-right
+  # - 4column-left
+  # - 4column-right
+  - column
+  # - row
+  # - floating
+  # - widescreen-tall
+  # - widescreen-tall-right
+  # - bsp
+
+#	First mod (default option + shift).
 mod1:
-    - option
-    - shift
-    - command
+  - option
+  - shift
+  # - control
+  # - command
+
+# Second mod (default option + shift + control).
 mod2:
-    - option
-    - shift
-    - control
-    - command
+  - option
+  - shift
+  - control
+  # - command
+
+# Commands:
+# special key values
+# "space"
+# "enter"
+# "up"
+# "right"
+# "down"
+# "left"
```

Comment:
AFAIK YAML requires quotes only for the special characters below:
```
: { } [ ] , & * # ? | - < > = ! % @ `
```

Just a matter of style and preferences I would use the below unquoted.

I would add `,` `.` as these are special characters implemented by default. And I would use single quote as best practice. (i.e. `"\"` breaks the config, but `'\'` doesn't. However, a simple `key: \` works too, as it is not considered a special character...)

```suggestion
# space
# enter
# up
# right
# down
# left

# special characters require quotes
# '.'
# ','
```